App/Modules/Slug.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Modules;
+
+use Cocur\Slugify\Slugify;
+use Core\Modules\Module;
+
+class Slug extends Module{
+    public function isSlugValid(string $slug):bool
+    {
+        $slugify = new Slugify();
+        $validSlug = $slugify->slugify($slug);
+
+        return $slug === $validSlug;
+
```

Comment:
empty line

Reply:
Resolved in branch Security_and_bug_fixes